scripts/core/device.yml

```diff
@@ -11,6 +11,73 @@ type: header
 desc: "Intel $OneApi Unified Runtime APIs for Device"
 ordinal: "2"
 --- #--------------------------------------------------------------------------
+type: macro
+desc: |
+      Target identification strings for $x_device_binary_t.pDeviceTargetSpec 
+      A device type represented by a particular target triple requires specific 
+      binary images. We need to map the image type onto the device target triple
+name: "$X_DEVICE_BINARY_TARGET_UNKNOWN"
+value: "\"<unknown>\""
+--- #--------------------------------------------------------------------------
+type: macro
+desc: |
+     SPIR-V 32-bit image <-> "spir", 32-bit OpenCL device
```

Comment:
I would hesitate to name-drop OpenCL in here, still not sure what we're going to do about #10. Could say something like "32-bit address width"

Reply:
I'm guessing this was lifted from PI?

Reply:
This is essentially copy pasta from PI.

Reply:
alright I'll add a note to #10 anyway